### DISCUSSION TITLE
Get media player status to HA

### DISCUSSION THREAD

**Discussion by marcfager** [music-assistant/support#2294] [Q&A] [answered]
Hi,

Thank you for an awesome project!

I have a Hifiberry integrated via UPnP/DLNA to Music assistant. The Hifiberry is connected to an amplifier, which must be turned on/off via IR commands (to which I use a small device with esphome). Is it possible any way to pick up the signal from Music Assistant when I "power on" a player to HA? I would like to use this to turn on the amplifier. I can pick up info once the player is playing music, but then I miss the first seconds.

**Comment by Rudd-O** [comment]
I think the most expedient way to solve your particular use case would be to use GPIO in the berry to signal to the ESPHome device to send the wireless command, no HA involved.  Make a digital binary sensor in ESPHome and write small scripts on the binary sensor that turn the amp on via IR when the sensor goes high and off when the sensor goes low.

**Reply by OzGav** [reply]
What about an automation that monitors the playing changing from off to anything else. I turn my amp on via IR when my TV turns on this way

**Comment by OzGav** [comment, marked as answer]
From MA 2.4 there is an option to map a HA entity to the MA player. So you can map your IR commands script or whatever. https://beta.music-assistant.io/player-support/#player-controls
